superduperdb/backends/dask/compute.py

```diff
@@ -40,7 +39,8 @@ def __init__(
                 **kwargs,
             )
 
-        logging.info("Compute Client is ready.", self.client)
+    def name(self) -> str:
+        return "Dask"
```

Comment:
u can return the client name

scheduler: uri

Reply:
Removed the line because it interferes with `test_cli`.